pvlib/modelchain.py

```diff
@@ -1041,6 +1043,28 @@ def faiman_temp(self):
     def fuentes_temp(self):
         return self._set_celltemp(self.system.fuentes_celltemp)
 
+    @staticmethod
+    def _irrad_for_celltemp(total_irrad, effective_irradiance):
+        """
+        Determine irradiance to use for cell temperature models, in order
+        of preference 'poa_global' or 'effective_irradiance'
+
+        Returns
+        -------
+        None.
```

Comment:
not accurate

Reply:
> > One question for discussion: when the code switches to use effective irradiance vs. 'poa_global' should the user be warned?
> 
> I lean no.

I also, but the switch should be made clear in the `ModelChain` documentation, and in the docstring for `ModelChain.run_model_from_effective_irradiance`